data/distrodefs/fedora/imagetypes.yaml

```diff
@@ -1574,7 +1574,20 @@ image_types:
       - "bootiso"
     exports: ["bootiso"]
     required_partition_sizes: *default_required_dir_sizes
-    installer_config: *default_installer_config
+    installer_config: 
+      <<: *default_installer_config
+      # for some reason the live installer didn't have the additional dracut modules, this
+      # might be a bug, reset them to empty?
```

Comment:
I'd remove the implied uncertainty and spell it out clearer here.  Like
```
For some reason, the live-installer never had or never took into account the
additional dracut modules. This might be a bug but, for now, we reset them to
empty to avoid changing the live-installer definition.
```

Or something to that effect.